docs/sensors/filters/ctx.md

```diff
@@ -0,0 +1,84 @@
+
+# Context Filter
+
+Context filter is applied to the event context. A CloudEvent from Webhook event-source has payload structure as:
+
+```json
+{
+    "context": {
+      "type": "type_of_event_source",
+      "specversion": "cloud_events_version",
+      "source": "name_of_the_event_source",
+      "id": "unique_event_id",
+      "time": "event_time",
+      "datacontenttype": "type_of_data",
+      "subject": "name_of_the_configuration_within_event_source"
+    },
+    "data": {
+      "header": {},
+      "body": {},
+    }
+}
+```
+
+## Fields
+
+Context filter has following fields:
+
+```yaml
+filters:
+  context:
+    type: event_type
+    subject: event_type
+    source: event_source
+    datacontenttype: event_type
```

Comment:
This is always `application/json`. Can remove it from the doc I think.

Reply:
Overall I think the context filter play less important role right now, because `source` (eventSourceName) and `subject` (eventName) have already been specified in the dependency, so specifying them again in the context filter makes no sense. 

But this will be useful if we have our new eventbus implementaion by using JetStream, which might be able to enable us to specify multiple eventSourceNames or eventNames (with regex).

Reply:
> Super helpful. Some basic comments that you could address but not biggie.
> 
> I was having a bit of trouble with data filters (actually need to go look at that code as it seems to allow partial matches of a string, will look at that out of this PR though)
> 
> So i switched to an `expr` after reading this, and my problem is solved. So you've helped at least 1 person today :D !!

Great!! It's really a pity that filtering section is hidden and not well documented, so here the PR :)

Reply:
I see, but I was reporting what exactly is checked in `sensors/dependencies/filter.go` in the function `filterContext`.
If such checks don't make sense, then we should simply remove them from doc but also from the code.

For now I suggest to keep everything in code and doc, specifying that:
- `datacontenttype` is always `application/json`
- `source` corresponds to `eventSourceName` specified in the Sensor YAML manifest
- `subject` corresponds to `eventName` specified in the Sensor YAML manifest
Then once the EventSource filtering will be implemented, we can simply move this piece of doc to that section and remove "useless code" from filterContext function.